### DISCUSSION TITLE
Update EMS-ESP-Loader

### DISCUSSION THREAD

**Discussion by proddy** [emsesp/EMS-ESP32#1276] [General]
@MichaelDvP shall we update the base code in  https://github.com/emsesp/EMS-ESP-Loader to 3.6.0 ?

**Comment by MichaelDvP** [comment]
That's a lot of work. With alova and yarn/vite it will shrink the image, but need a lot of manual work, cpp is not so much change.

**Reply by proddy** [reply]
hmm, ok

**Comment by bbqkees** [comment]
Maybe it's good to leave it as is because then it is more obvious that the loader is different than the actual EMS-ESP.
The loader, although soon necessary for the 4MB modules, can be quite a challenge to understand for the non-technical people.

**Reply by proddy** [reply]
agree, too much work to change and I have no time
